tests/test_on_failure.py

```diff
@@ -11,7 +11,7 @@
         "call": {
             "forward": {
                 "name": "status",
-                "reply": {"ok": {"type": "bool"}},
+                "reply": {"type": {"ok": "bool"}},
```

Comment:
For completeness we might want to test the `value` and `identical` checks too.

Reply:
that's done in test_checks.py

Reply:
Oh I forgot to add that file.